components/brave_vpn/brave_vpn_os_connection_api_sim.cc

```diff
@@ -84,6 +92,12 @@ void BraveVPNOSConnectionAPISim::OnCreated(const std::string& name,
 
 void BraveVPNOSConnectionAPISim::OnConnected(const std::string& name,
                                              bool success) {
+  // Cancelling connecting request simulation.
```

Comment:
Not clear what you mean with this comment. Is the only time `disconnect_requested_` true when the `--brave-vpn-simulate` argument is passed?

Reply:
oh wait - I see now 😂  This whole file is the simulation